nuxeo-core/nuxeo-core-bulk/src/test/java/org/nxueo/ecm/core/scroll/TestScrollService.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     bdelbosc
+ */
+package org.nxueo.ecm.core.scroll;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.core.api.scroll.Scroll;
+import org.nuxeo.ecm.core.api.scroll.ScrollRequest;
+import org.nuxeo.ecm.core.api.scroll.ScrollService;
+import org.nuxeo.ecm.core.bulk.CoreBulkFeature;
+import org.nuxeo.ecm.core.scroll.StaticScrollRequest;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+@RunWith(FeaturesRunner.class)
+@Features(CoreBulkFeature.class)
+public class TestScrollService {
+
+    @Inject
+    protected ScrollService service;
+
+    @Test
+    public void testScrollService() {
+        assertNotNull(service);
+        assertTrue(service.exists("default"));
+        assertTrue(service.exists("static"));
+        assertTrue(service.exists("repository"));
+        assertFalse(service.exists("unknown"));
```

Comment:
i am wondering if we add a new tests or even actually we are not sure about the execution order of method. Your idea about this test is to ensure that the all is correctly configured right ? what if this test is an `@Before` or `BeforeTest` and the test:

```
 @BeforeClass
    public static void testScrollService() {
        final ScrollService service = Framework.getService(ScrollService.class);
        assertNotNull(service);
        assertTrue(service.exists("default"));
        assertTrue(service.exists("static"));
        assertTrue(service.exists("repository"));
        assertFalse(service.exists("unknown"));
    }
```

i agree we don't test the injection here by `BeforeClass` but this the responsibility  of `FeaturesRunner` and its tests .

Reply:
Maybe, but I like simple and dumb unit test, this test checks that the service is started and well initialized, we don't need to test that on all tests. I don't see cases where a Before something is useful to understand failure in this case.